google/cloud/storage/_http.py

```diff
@@ -71,7 +71,7 @@ def api_request(self, *args, **kwargs):
         retry = kwargs.pop("retry", None)
         headers = {
             **kwargs.pop("headers", {}),
-            "x-goog-api-client": self._connection.user_agent + " gccl-invocation-id: " + uuid4(),
+            "x-goog-api-client": self._client_info.user_agent + " gccl-invocation-id: " + uuid4(),
```

Comment:
Two things:

1. it should be `gccl-invocation-id/UUID`
2. introduce an option to disable this invocation ID from being added

Reply:
Ah yep, 1 should be updated now.
And cool, will work on 2. Trying to figure out testing now, going to add my current quandary in another comment.

Reply:
After discussion with @frankyn and @BenWhitehead, it was decided that for the moment, we will not be introducing an option to disable the invocation id.